distribution/client/src/lib/supervisord.ts

```diff
@@ -50,7 +50,7 @@ export default class Supervisord {
   static startProcess(name) {
     logger.info(`[supervisord] Starting ${name} process`);
     return new Promise((resolve, reject) => {
-      client.methodCall('supervisor.startProcess', [name], (e, value) => {
+      client.methodCall('supervisor.startProcess', [name,true], (e, value) => {
```

Comment:
:+1: 

In case anybody else is looking, this is telling [supervisord to wait](http://supervisord.org/api.html#supervisor.rpcinterface.SupervisorNamespaceRPCInterface.startProcess)

Reply:
FTR, associated dedicated commit for this is https://github.com/jenkins-infra/evergreen/pull/321/commits/79a5eb02d933e023b737f6ecae671b21e5c28170